packages/block-library/src/navigation-menu-item/editor.scss

```diff
@@ -82,9 +81,9 @@
 
 	&.is-editing,
 	&.is-selected {
-		box-shadow: 0 0 1px $light-gray-900 inset;
-		border-radius: 4px;
 		min-width: 30px;
+		font-family: inherit;
+		font-size: $text-editor-font-size;
```

Comment:
Does this need to be specified? Seems weird since a theme can change the size.

Reply:
no, I removed it from here and from above this in the parent rule.

Reply:
no, I removed it